### DISCUSSION TITLE
What word describes a broadcaster, commentator, or writer who specializes in political analysis?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#482] [Galaxy Trivia] [answered]
**Hint:** Derived from a Hindi phrase meaning “learned”.

**Comment by ParanoidUser** [comment]
@KR-Ravindra, @DrakonZZZ, @mayankjaiswal1709, @muzammilali28, anyone? 😶‍🌫️

**Comment by ParanoidUser** [comment]
@PARKER-X, @iamharshil, @Sanjaypranav, @Mritunjaysri01, maybe you can answer? 😏

**Reply by PARKER-X** [reply]
Pandit @ParanoidUser

**Reply by PARKER-X** [reply]
Sorry pooondit

**Reply by PARKER-X** [reply]
Pundit

**Reply by ParanoidUser** [reply]
Could you post it as a separate message? There is no way to mark the reply as an answer. 😊

**Comment by mayankjaiswal1709** [comment, marked as answer]
Thank you for the hint! In that case, the word you are thinking of is likely "pundit", which indeed comes from the Hindi word "pandit" (पण्डित), meaning "learned". The term "pundit" is often used in English to refer to an expert or authority in a particular field, especially politics.

**Comment by PARKER-X** [comment]
Pundit

**Comment by thuongtruong1009** [comment]
A pundit is a person who offers [mass media](https://en.wikipedia.org/wiki/Mass_media) opinion or commentary on a particular subject area (most typically politics, the social sciences, technology or sport). The term originates from the [Sanskrit](https://en.wikipedia.org/wiki/Sanskrit) term [pandit](https://en.wikipedia.org/wiki/Pandit) ([paṇḍitá](https://en.wiktionary.org/wiki/%E0%A4%AA%E0%A4%A3%E0%A5%8D%E0%A4%A1%E0%A4%BF%E0%A4%A4#Sanskrit) पण्डित), meaning "knowledge owner" or "learned man".[[1]](https://en.wikipedia.org/wiki/Pundit#cite_note-EB1911-1) It refers to someone who is erudite in various subjects and who conducts religious ceremonies and offers counsel to the king and usually referred to a person from the [Hindu](https://en.wikipedia.org/wiki/Hindu) [Brahmin](https://en.wikipedia.org/wiki/Brahmin) but may also refer to the [siddhas](https://en.wikipedia.org/wiki/Siddhas), [Siddhars](https://en.wikipedia.org/wiki/Siddhar), [Naths](https://en.wikipedia.org/wiki/Nath), [ascetics](https://en.wikipedia.org/wiki/Ascetics), [sadhus](https://en.wikipedia.org/wiki/Sadhus), or [yogis](https://en.wikipedia.org/wiki/Yogis) ([rishi](https://en.wikipedia.org/wiki/Rishi)).

**Comment by Sanjaypranav** [comment]
i'm an Indian but idk hindi we call them "Arasiyal Aaivalar (அரசியல் ஆய்வாளர்) " in tamil
